### ISSUE TITLE
403 and 404 not found errors

### ISSUE THREAD

**Issue by guntel** [ivandiazwm/phpback#131]
After a fresh install into mydomain.net/feedback I am getting a 403 error after the install finishes, however if I remove the .htaccess, it displays the index, however the login page throws a 404 error. I've read through issues but only have found IIS related issues like mine.

**Comment by SirDev** [comment]
@guntel .htaccess files don't work on IIS, you'll need to set up redirects there through a web.config file in the root of the site. I'd urge you to try the automatic converter at http://cbsa.com.br/tools/online-convert-htaccess-to-web-config.aspx

**Comment by hi-nrg-joe** [comment]
I think it's setting on the webserver not allowing it to work correctly. I struggled with this for a while wondering if I need to sym link them to the directory, or copy the directory, etc....I ended up having to edit the httpd.conf file and change the htaccess section. Instead of "AllowOveride None" change it to "AllowOveride All", restart the webserver and it worked just fine.
I'm not sure how to fix the same issue with IIS as I don't use it, but the above solution worked with my Apache server.
